src/theme/createTheme.ts

```diff
@@ -37,7 +37,7 @@ export const createTheme: (palette: IPalette) => ITheme = (palette) => {
         unclesBoxText: BLUE,
         txBoxText: WHITE,
         cmBoxText: WHITE,
-        txViewSelector: DAWN_D5,
+        txViewSelector: DUSK,
```

Comment:
See if it makes sense to remove txViewSelector and txViewSelectorActive and replace with colors.base.*